CMakeLists.txt

```diff
@@ -14,7 +13,16 @@ enable_testing()
 include(SetupGTest)
 
 # C++ version
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+if ("${CMAKE_VERSION}" LESS "3.1")
```

Comment:
Yup, that seems correct, I didn't realize this at the time! Thanks! That probably would have hit us in the future.

We probably didn't see this so far because we've only been looking for versions that are older. I'll fix this in a bit!

Thanks for taking a look at this! :smile: